_pages/manage/annual-assessment/2023-03-28-manage-gwaa.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: page
+sidenav: true
+permalink: manage/section-508-assessment/
+type: manage
+title: "Governmentwide Section 508 Assessment"
+redirect_from:
+- section-508-assessment/
```

Comment:
Need to add another redirect reference from:
- section-508-assessment/annual-reports/